demisto_sdk/commands/pre_commit/hooks/format.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+from typing import Iterable, Optional
+
+from demisto_sdk.commands.pre_commit.hooks.hook import Hook
```

Comment:
@ilaner please document the behaviour of the `[-1]` why the hack? and i would consider creating a common class as both the FormatHook and ValidateHook share the exact same logic.

Reply:
I combined both of the hooks to the same class as they are the same for now.
Basically, all demisto-sdk hooks should be the same as they follow the same API (-i or -g).

In addition I added a comment:
We do it because the default parameter is `-g` (run on changed files). If we want to run on input files, we should modify the `-g` parameter and append the input parameter.
We decided to have all the default values of the pre-commit in the config file instead of setting them in the code. In the code we just modify for non-default scenarios.